1_assignment/alphbet_frequency.js

```diff
@@ -16,28 +21,30 @@ function calculateFrequency(string) {
   return frequency;
 }
 
-function calculateFrequency2(string) {
+
+
+// Solution 2  : using only Arrays and not using String functions.
+
+function calculateFrequencyUsingArrays(string) {
   // Write your code here
   var i=0;
   var j=0;
-  frequency=new Array(26).fill(0);
-  alphabet_array=["a" , "b" ,"c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z"]
-  for (i =0; i< string.length ; i++)
+  var frequency=new Array(26).fill(0);
+  var alphabet_array=["a" , "b" ,"c","d","e","f","g","h","i","j","k","l","m","n","o","p","q","r","s","t","u","v","w","x","y","z"];
+  for (i=0 ; i < string.length ; i++)
   {
-  	for (j=0; j < 26 ;j++)
+  	for (j = 0; j < 26 ; j++)
   	{
-
-
-  		{
+      {
```

Comment:
Same line opening braces, too much indentation
Rule of thumb:
1. Always indent using spaces
2. A single indent should be of 2 spaces.
Read about coding styles, follow one and be extremely consistent with it